### DISCUSSION TITLE
How am i supposed to get past CORS in google sites

### DISCUSSION THREAD

**Discussion by jmazzgamerpro** [ruffle-rs/ruffle#18271] [Q&A]
So recently i've been working on a site where you can play all of the Papa's games, but none will work due to "Ruffle failed to load the flash swf file. Access to fetch has likely been blocked by CORS policy." I have no idea what this means and i have no idea how to get any of these to work on my site. I have little experience with programming, so what is the easiest way to fix this on google sites?

**Comment by Croworbit** [comment]
CORS policy applies if you try and get resources from another site and link them to yours

you basically need to have whatever site you're pulling say, a swf from, allow you to do so

https://en.m.wikipedia.org/wiki/Cross-origin_resource_sharing

you used to be able to not care about CORS using flash player, I think because CORS came after

ruffle just can't ignore it, so you'll need to host your own resources on a seperate server, or get permission from someone else

**Reply by jmazzgamerpro** [reply]
Ok, i'll try hosting the .swfs on a different server.

**Reply by jmazzgamerpro** [reply]
Ok so i've put the files i need on a github repository, how do i enable the "access-control-allow-origin" header?

**Reply by n0samu** [reply]
@jmazzgamerpro You need to create a GitHub Pages repo and use a github.io link to access your files. See https://docs.github.com/en/pages/getting-started-with-github-pages/creating-a-github-pages-site

**Reply by foxagen** [reply]
Hey have you fixed this issue? I've been trying to do the same thing but can't figure out what I'm supposed to do to fix it.

**Reply by n0samu** [reply]
@foxagen See https://github.com/ruffle-rs/ruffle/wiki/Using-Ruffle#configure-cors-header

**Reply by Croworbit** [reply]
> Hey have you fixed this issue? I've been trying to do the same thing but can't figure out what I'm supposed to do to fix it.

yes, because the issue is not with ruffle
